src/cli.py

```diff
@@ -40,7 +37,7 @@ def parse_cli_args():
 
     output_group: argparse._MutuallyExclusiveGroup = (
         parser.add_mutually_exclusive_group()
-    )  # pylint: disable=protected-access
+    )
```

Comment:
I enabled https://pre-commit.ci Much simpler to get these as PRs than have to go in manually afterwards

below change is just to poke CI

```suggestion
                                )
```

Reply:
Wow, that is incredibly helpful, thanks